appserver/extras/docker-images/micro/src/main/docker/Dockerfile

```diff
@@ -4,20 +4,21 @@ FROM @docker.java.image@
 EXPOSE 6900 8080
 
 # Configure environment variables
-ENV PAYARA_HOME=/opt/payara\
+ENV HOME_DIR=/opt/payara
```

Comment:
As in other Dockerfiles, despite it uses same directory here

Reply:
The change of the ENV name will affect all users who extend our images and possible require them to adjust their dockerfiles.

Reply:
In this release is much more breaking changes in docker files, so I think it is a good oportunity to synchronize behavior of all images to same principles and provide documentation about how things work.
But then it would be good to use also same directory structure in all three Dockerfiles.

Reply:
Seems that other changes are not so much breaking as I thought, so I brought  the PAYARA_HOME back to life here.